pageObject/summaryPage.js

```diff
@@ -9,6 +9,9 @@ let cartTitleTextBoxLocator = by.css('#cart_title');
 let quantityPlusButtonLocator = by.css('.button-plus');
 let quantityInPutLocator = by.css('.cart_quantity_input.form-control.grey');
 let proceedToCheckoutButtonLocator = by.css('.standard-checkout.button-medium');
+let deleteLastCartItemButtonLocator = by.css('.last_item a[class="cart_quantity_delete"]');
+let homeButtonLocator = by.css('a.home');
+let lastCartProductTextBoxtLocator = by.css('[id="product_3_13_0_0"]');
```

Comment:
is this selector always correct for the last item? for example if we have 3 items or we have 5 items in cart

Reply:
@germange What about this place